TypeApprovalRegister+services.yaml

```diff
@@ -124,17 +123,17 @@ clients:
           - service-name: /v1/relay-operation-update
             uuid: tar-1-0-0-op-c-0023
         individual:
-      service:
-        basic:
-        individual:
           - service-name: /v1/inquire-application-type-approvals
```

Comment:
since uuid of /v1/register-application, /v1/relay-server-replacement, /v1/deregister-application, /v1/relay-operation-update of RegistryOffice uuid are updated in other files, we can also update them in services.yaml file

Reply:
We will establish a process for updating all UUIDs soon.

Reply:
Sure, we can update uuid(s) in that step